prisma/seed.ts

```diff
@@ -1,69 +1,26 @@
-import { connect } from "http2";
-import { prisma } from "../lib/db";
-import { User, Prisma } from "@prisma/client";
+import { PrismaClient } from "@prisma/client";
 
-async function main() {
-  await prisma.user.deleteMany();
-  await prisma.bookmark.deleteMany();
+const prisma = new PrismaClient();
 
-  const users: Prisma.UserCreateInput[] = [
-    {
-      email: "dan@devhausleipzig.de",
-      firstName: "Dan",
-      lastName: "McAtee",
-      password: "test123",
-    },
-    {
-      email: "taylor@devhausleipzig.de",
-      firstName: "Taylor",
-      lastName: "Harvey",
-      password: "test123",
-    },
-    {
-      email: "franz@devhausleipzig.de",
-      firstName: "Franz",
-      lastName: "Wollang",
-      password: "test123",
-    },
-    {
-      email: "nikita@devhausleipzig.de",
-      firstName: "Nikita",
-      lastName: "Nakropin",
-      password: "test123",
-    },
-  ];
-  const createdUsers = [];
-  for (const user of users) {
-    const createdUser = await prisma.user.create({ data: user });
-    createdUsers.push(createdUser);
-  }
+async function main() {
+  const seatIds = ["A1", "A2", "A3", "B1", "B2", "B3", "C1", "C2", "C3"];
 
-  const bookmarks: Prisma.BookmarkCreateInput[] = [
-    {
-      movieId: "653346",
-      user: { connect: { id: createdUsers[0].id } },
-    },
-    {
-      movieId: "653346",
-      user: { connect: { id: createdUsers[2].id } },
-    },
-    {
-      movieId: "653346",
-      user: { connect: { id: createdUsers[0].id } },
-    },
-    {
-      movieId: "693134",
-      user: { connect: { id: createdUsers[1].id } },
-    },
-    {
-      movieId: "693134",
-      user: { connect: { id: createdUsers[0].id } },
-    },
-  ];
+  await Promise.all(
```

Comment:
Please loop over seats with a for-loop and then await prisma.seat.create(...) . Just like with users in the main branch.


Please also seed ScreeningSeats.